src/components/queryBuilder/FilterEditor.tsx

```diff
@@ -362,12 +362,12 @@ export const FilterEditor = (props: {
 };
 
 export const FiltersEditor = (props: {
-  fieldsList: FullField[];
+  allColumns: TableColumn[];
   filters: Filter[];
   onFiltersChange: (filters: Filter[]) => void;
 }) => {
-  const { filters = [], onFiltersChange, fieldsList = [] } = props;
-  const { label, tooltip, AddLabel, RemoveLabel } = selectors.components.QueryEditor.QueryBuilder.WHERE;
+  const { filters = [], onFiltersChange, allColumns: fieldsList = [] } = props;
+  const { label, tooltip, addLabel } = labels.components.FilterEditor;
```

Comment:
here and in other places: immutable values should be declared outside of the React components